components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/client/ApplicationManagementServiceClient.java

```diff
@@ -82,7 +83,9 @@ public ApplicationManagementServiceClient(String cookie, String backendServerURL
      * @param serviceProvider
      * @throws AxisFault
      */
+    @Deprecated
```

Comment:
why this is deprecated?

Reply:
Since this method will not get executed in client side, have removed it. Included a new method "createApplicationWithTemplate(ServiceProvider serviceProvider, String templateName) " instead of that.

Reply:
Since this method will not get executed in client side, have removed it considering that ApplicationManagementServiceClient is not customizable. Included a new method "createApplicationWithTemplate(ServiceProvider serviceProvider, String templateName) " instead of that.